examples_whitebox_test.go

```diff
@@ -1,23 +1,26 @@
 package goforit_test
 
 import (
+	"context"
 	"fmt"
 	"time"
 
 	"github.com/stripe/goforit"
 )
 
 func Example() {
+	ctx := context.Background()
+
 	// flags.csv contains comma-separated flag names and sample rates.
 	// See: fixtures/flags_example.csv
 	backend := goforit.BackendFromFile("flags.csv")
 	goforit.Init(30*time.Second, backend)
 
-	if goforit.Enabled("go.sun.mercury") {
+	if goforit.Enabled(ctx, "go.sun.mercury") {
 		fmt.Println("The go.sun.mercury feature is enabled for 100% of requests")
 	}
 
-	if goforit.Enabled("go.stars.money") {
+	if goforit.Enabled(ctx, "go.stars.money") {
```

Comment:
Let's also add an example of using this without context (since passing in a nil context should be a safe operation).